app/controllers/games_controller.rb

```diff
@@ -23,14 +23,22 @@ def create
 
   def update
     @game = Game.find(params[:id])
-    if @game.black_player_id != current_user
-      @game.update_attribute(:black_player_id, current_user.id)
+    if @game.white_player_id != current_user.id && @game.black_player_id != current_user.id 
+      @game.update_attribute(:black_player_id, current_user.id) 
       redirect_back(fallback_location: root_path, alert: 'You have joined the game as the Black player.')
     else
       redirect_back(fallback_location: root_path, alert: 'You have already joined this game.')
     end
   end
 
+  def destroy
+    @game = Game.find(params[:id])
+    return render_not_found if @game.blank?
+    @game.update_attribute(:status, "complete")
+    @game.destroy
+    redirect_back(fallback_location: root_path, alert: 'Player has forfeited, game over!')
+  end
+
```

Comment:
This destroy method is used for our Concede button. It will probably change slightly once I get help with the Concede logic, but for now, it works as intended.